src/plugins/dataflow/model/utilities/node.ts

```diff
@@ -172,6 +172,30 @@ export const NodeOperationTypes = [
     numberSentence: (n1: string, n2: string) => `ceil(${n1}) = `,
     icon: EmgIcon
   },
+  {
+    name: "Ramp",
+    type: "transform",
+    method: (n1: number, n2: number, tMinus1?: number) => {
+      if (!tMinus1 || !isFinite(tMinus1)) return n1;
```

Comment:
It seems like `0` could be a valid value for `tMinus1` which would be excluded by this test. If you just want to test for valid numbers you can use `isFiniteNumber()` in `math-utils.ts`.